pkg/frontend/authenticate.go

```diff
@@ -543,12 +543,13 @@ var (
 		"system_metrics":     0,
 	}
 	sysWantedTables = map[string]int8{
-		"mo_user":       0,
-		"mo_account":    0,
-		"mo_role":       0,
-		"mo_user_grant": 0,
-		"mo_role_grant": 0,
-		"mo_role_privs": 0,
+		"mo_user":                 0,
+		"mo_account":              0,
+		"mo_role":                 0,
+		"mo_user_grant":           0,
+		"mo_role_grant":           0,
+		"mo_role_privs":           0,
+		"%!%mo_increment_columns": 0,
```

Comment:
What is this?   "%!%", should that be some well documented way of generate this weird thing?

Reply:
I think it just wants to distinguish other names. Maybe @jianwan0214 has a more detailed description. This pr will be turned off，since it takes too long to get in. pr 5237 will fix this problem.

Reply:
it is the auto_incr table name to store the increment col value. This strange name is to avoid the exist table name.